### ISSUE TITLE
Text size should match Atom editor window

### ISSUE THREAD

**Issue by pcgeek86** [jeremyramin/terminal-plus#95]
The text size of the terminal sessions should match the text size of the Atom Editor window, for consistency. Right now, it doesn't match up. Using Atom 1.2.0 on Mac OS X 10.11.1.

<img width="737" alt="screen shot 2015-11-16 at 8 48 13 am" src="https://cloud.githubusercontent.com/assets/466713/11186424/d9376a00-8c3e-11e5-8a1b-78883170cc65.png">

Cheers,
Trevor Sullivan
Microsoft MVP: PowerShell
http://trevorsullivan.net
http://twitter.com/pcgeek86

**Comment by jeremyramin** [comment]
This can be added.

**Comment by jeremyramin** [comment]
Added in v0.14.3

**Comment by pcgeek86** [comment]
@jeremyramin Awesome!! Thanks!

Cheers,
Trevor Sullivan

**Comment by nunocastromartins** [comment]
By the way, on a similar note.
Shouldn't the font, by default, also match the font of the editor window?

**Comment by jeremyramin** [comment]
@pcgeek86 No problem.

@nunocastromartins Actually the terminal needs a different font (specifically a monospace font) in order to function properly.

**Comment by nunocastromartins** [comment]
What do you mean by different font? The editor's font family should already be monospaced, no?
Do people program in non-monospaced fonts?

**Comment by pcgeek86** [comment]
@jeremyramin My suggestion would be to enable that consistency with the editor window, and offer an override in the settings. Personally, I don't see myself ever changing the default font in Atom Editor or the Terminal-Plus window, but that's just me. If, for some very odd reason, I did end up changing the Atom font, I would kinda naturally expect the Terminal-Plus window to match it.

As far as the implementation goes, I envision a drop-down box with a font list, in the settings. At the very top of that combo box, there would be a default `<inherit>` option.

Cheers,
Trevor Sullivan
Microsoft MVP: PowerShell

**Comment by nunocastromartins** [comment]
@jeremyramin I agree that it isn't much of an issue since it is still possible to manually set it so that both fonts match.
I was just pointing out that, by default, the terminal's font _should_ be the same as the text editor's (for the same reason that the text size should be the same - consistency).

If, by any chance, the editor's font isn't monospaced what exactly goes wrong (besides having misaligned outputs such as when running `ls -l`)? Is there some bigger issue, perhaps related with the dimensions of the terminal window (just guessing, here)?
In any case, the user can still change the font to a monospaced one in the settings.

I realise that this is just me being quite picky, I only mentioned it because when I first installed this extension I found it weird that the font looked different from the editor's (in my case it looked bad, because the default font doesn't render very nicely in my computer) - I quickly figured out that it was possible to change it, though. :smile:
